### ISSUE TITLE
Classification losses

### ISSUE THREAD

**Issue by wagenaartje** [neural-fit/neuralfit#3]
Although classification is currently possible, there are no supporting loss functions for it and output neurons cannot be constrained. Therefore, NeuralFit should

- Add loss functions for classification (e.g. [cross entropy](https://keras.io/api/losses/probabilistic_losses/#categoricalcrossentropy-class)).

Feel free to make any additions below!
